string audio_topic_name
string image_topic_name
int32 queue_size
string file_name # This will be used as key
string file_format
string audio_format
string audio_sample_format
int32 audio_channels
int32 audio_depth
int32 audio_sample_rate
string video_encoding
int32 video_height
int32 video_width
int32 video_framerate
